src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -100,7 +100,7 @@ public static bool VerifyWithECDsa(byte[] message, byte[] pubkey, byte[] signatu
         }
 
         // This is for solving the hardfork issue in https://github.com/neo-project/neo/pull/3209
-        [ContractMethod(true, Hardfork.HF_Cockatrice, CpuFee = 1 << 15, Name = "verifyWithECDsa")]
+        [ContractMethod(true, Hardfork.HF_Cockatrice, CpuFee = 1 << 15, Name = nameof(VerifyWithECDsa))]
```

Comment:
i have a question that is not related to this pr, can we rename the CPUFee to system fee?

Reply:
No. There is `CpuFee`. And there is `StorageFee`. They're very different even though system fee pays for everything in the end.